File: 01 Introdução/2165 Tuitando/2165.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  string name;
  getline(cin, name);

  if (name.length() <= 140)
  {
    cout << "TWEET" << endl;
  }
  else
  {
    cout << "MUTE" << endl;
  }
  return 0;
}

// https://www.beecrowd.com.br/judge/pt/problems/view/2165
// Ranking 02827º, Time 0.000, Submiss Date 01/04/2023 21:15:13 (C++17)

File: 01 Introdução/2165 Tuitando/2165.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String name = sc.nextLine();
    if (140 >= name.length()) {
      System.out.println("TWEET");
    } else {
      System.out.println("MUTE");
    }
    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/2165
// Ranking 00749º, Time 0.128, Submiss Date 01/04/2023 20:54:17 (Java 14)
